examples/crm/src/dataGenerator/dealNotes.ts

```diff
@@ -3,16 +3,13 @@ import { random, lorem } from 'faker/locale/en_US';
 import { Db } from './types';
 import { randomDate } from './utils';
 
-const type = ['Email', 'Call', 'Call', 'Call', 'Call', 'Meeting', 'Reminder'];
-
 export const generateDealNotes = (db: Db) => {
     return Array.from(Array(300).keys()).map(id => {
         const deal = random.arrayElement(db.deals);
         deal.nb_notes++;
         return {
             id,
             deal_id: deal.id,
-            type: random.arrayElement(type),
```

Comment:
Why did you remove this ?

Reply:
None of the note components display or use this property. Or maybe I've missed it